common.go

```diff
@@ -36,6 +36,14 @@ func GetLanguageExtensions(language string) []string {
 
 // GetLanguage return the Language for a given filename and file content.
 func GetLanguage(filename string, content []byte) string {
+	if lang, safe := GetLanguageByFilename(filename); safe {
+		return lang
+	}
+
+	if lang, safe := GetLanguageByShebang(content); safe {
```

Comment:
Isn't this and the rest of [shebang strategy](https://github.com/src-d/simple-linguist/pull/23/files#diff-a9c3f74e11f5aefc6762b6c5f6e78401R19) supposed to be part of #22 ?